mithril-aggregator/src/event_store/database/repository.rs

```diff
@@ -0,0 +1,61 @@
+//! Migration module
+//!
+use anyhow::anyhow;
+use mithril_common::StdResult;
+use mithril_persistence::sqlite::{ConnectionExtensions, SqliteConnection};
+
+use std::sync::Arc;
+
+use crate::event_store::database::query::InsertEventQuery;
+use crate::event_store::{event::Event, EventMessage};
+/// The EventPersister is the adapter to persist EventMessage turning them into
+/// Event.
+pub struct EventPersister {
+    connection: Arc<SqliteConnection>,
+}
+
+impl EventPersister {
+    /// Instantiate an EventPersister
+    pub fn new(connection: Arc<SqliteConnection>) -> Self {
+        Self { connection }
+    }
+
+    /// Save an EventMessage in the database.
+    pub fn persist(&self, message: EventMessage) -> StdResult<Event> {
+        let log_message = message.clone();
+        let mut rows = self.connection.fetch(InsertEventQuery::one(message)?)?;
+
+        rows.next().ok_or(anyhow!(
+            "No record from the database after I saved event message {log_message:?}"
```

Comment:
This error message was already present before, but I find the phrasing a bit strange with the use of 'I'. What do you think?
```suggestion
            "No record from the database after saving event message {log_message:?}"
```

Reply:
Done